educator/database/schema.sql

```diff
@@ -161,3 +165,22 @@ create table if not exists block_txs (
 );
 
 create index if not exists block_txs_blk_idx on block_txs (__idx);
+
+----------------------------------------------------------------------------
+-- Certificates
+----------------------------------------------------------------------------
+
+-- Creating 'certificates' table.
+--
+create table if not exists certificates (
+    hash  BYTEA  not null,
+    meta  JSONB  not null,
+    pdf   BYTEA  not null,
+
+    primary key (hash),
+
+);
+
+create index if not exists certificates_student_name
+    on certificates ((meta ->> 'studentName'));
```

Comment:
If we'd like to implement searching by diploma number too, we would also like to have `meta ->> 'number'` index. However, this is minor, since we'll do filtering on frontend anyway, at least initially.